crates/viewer/re_time_panel/tests/time_panel_tests.rs

```diff
@@ -55,27 +53,62 @@ pub fn time_panel_dense_data_should_match_snapshot() {
         rng_seed.wrapping_mul(0x2545_f491_4f6c_dd1d)
     };
 
-    let entity_path = EntityPath::from("/entity");
-    let mut builder = Chunk::builder(entity_path.clone());
-    for frame in 0..1_000 {
-        if rng() & 0b1 == 0 {
-            continue;
+    test_context.log_entity("/entity".into(), |mut builder| {
+        for frame in 0..1_000 {
+            if rng() & 0b1 == 0 {
+                continue;
+            }
+
+            builder = builder.with_sparse_component_batches(
+                RowId::new(),
+                [build_frame_nr(frame)],
+                [(MyPoint::descriptor(), Some(&points1 as _))],
+            );
         }
 
-        builder = builder.with_sparse_component_batches(
-            RowId::new(),
-            [build_frame_nr(frame)],
-            [(MyPoint::descriptor(), Some(&points1 as _))],
-        );
-    }
-    test_context
-        .recording_store
-        .add_chunk(&Arc::new(builder.build().unwrap()))
-        .unwrap();
+        builder
+    });
 
     run_time_panel_and_save_snapshot(test_context, TimePanel::default(), "time_panel_dense_data");
 }
 
+fn run_time_panel_and_save_snapshot(
```

Comment:
the difference between this an `run_expanded_time_panel_and_save_snapshot` isn't quite clear. both are exptended, one is just larger and expands the items rather than the time panel (as the name suggests). Looks like a single parameterized one would be better